sdk/python/foundation-models/system/inference/text-generation/llama-safe-online-deployment.ipynb

```diff
@@ -446,6 +446,23 @@
     "}"
    ]
   },
+  {
```

Comment:
can we also have a separate section for this? so customers ask, we can direct them with something like "refer to section 4.3". And it is more explicit.

Reply:
added